recipes/nanort/all/conanfile.py

```diff
@@ -0,0 +1,35 @@
+from conan import ConanFile
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
+
+
+import os
+
+
+class NanoRTConan(ConanFile):
+    name = "nanort"
+    description = "Single header only modern ray tracing kernel"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/lighttransport/nanort"
+    topics = ("graphics", "raytracing", "header-only")
+    license = "MIT"
+    package_type = "header-library"
+    settings = "os", "arch", "compiler", "build_type"
+    no_copy_source = True
+
```

Comment:
```suggestion

        @property
    def _min_cppstd(self):
        return 11

    def validate(self):
        if self.settings.compiler.get_safe("cppstd"):
            check_min_cppstd(self, self._min_cppstd)

```

The project needs C++11 so we have to validate that we are using the correct configuration